### DISCUSSION TITLE
A/Q: 2 GitHub Student Account!? Can save one for later?

### DISCUSSION THREAD

**Discussion by AliAryanTech** [community/community#45621] [GitHub Education] [answered]
**Explain:**
I know it's a childish question, I have 2 GitHub account with student package subscription, as one is my brother account which he made for me. I did request to GitHub for student package but coz of some stuff were missing in my profile it was not accepting it. So my brother did joined subscription from his account and give me, but later I got email from GitHub that my account is also verified, I mean now it have GitHub student package.

**Situation:**
So, here's my situation 🥴
2 github student account at the same time, and I don't want to waste it.

**Question:**
Is it possible to disable or inactive one account subscription for later use? Or it would stay if I don't use one, **"subscription expire in 1 year"**

**Comment by samuel-ma** [comment, marked as answer]
@AliAryanTech 
For as long as your a student, you can always apply again for the student developer pack after it expires, you don't need to worry about saving up an account😉

**Reply by AliAryanTech** [reply]
Thanks for informing
